hack/observability/kube-state-metrics/crd-config.yaml

```diff
@@ -0,0 +1,745 @@
+kind: CustomResourceStateMetrics
```

Comment:
Is it possible to add an header comment specifying this is an autogenerated file?

Reply:
Added a comment via the header file :-)